src/open_inwoner/scss/components/Button/Button.scss

```diff
@@ -214,7 +214,9 @@
 
 .h2 + .button,
 .h4 + .button,
-.p + .button {
+.p + .button,
+.utrecht-heading-2 + .button,
+.utrecht-heading-4 + .button {
```

Comment:
Should this set have a level 3?

Reply:
No, these are slightly annoying 'context' styles anyway, so the less, the better. 
Styles for margins should actually be set by the parent, not by the component itself (but... for content on pages it does feel a bit odd to call the page-content 'the parent'). 
So I've put these in here, as a copy of the 'legacy' styling.